.github/ISSUE_TEMPLATE/new_action.yml

```diff
@@ -12,3 +12,14 @@ body:
         Ex. It would be great to have an action that would ...
     validations:
       required: false
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Items that need to be completed along with the creation of the new App or Action (to be filled out later when the action is being developed)
+      options:
+        - label: Update [GitHub Actions section of README](https://github.com/github/github-ospo?tab=readme-ov-file#tools-created-by-github-ospo) linking to the new App or Action
+          required: true
```

Comment:
Will this require them to check the box before creating the issue?

Reply:
Yes. Can make false.